pydantic/decorators.py

```diff
@@ -168,6 +168,18 @@ def validator(
     mode: Literal['before', 'after'] = 'before' if pre is True else 'after'
 
     def dec(f: Any) -> _decorators.PydanticDescriptorProxy[Any]:
+        """
+        A decorator function for pydantic validation.
```

Comment:
Can someone else suggest "what it does" instead of "what it is" before merging?

Reply:
Actually, the description is already on the upper function. I think it's fine.

Reply:
Good callout, and a situation where the coverage tools are in opposition to linting and doc generation tools. I removed the unnecessary docstrings on the `dec` functions.